### ISSUE TITLE
Default authentication file is misplaced

### ISSUE THREAD

**Issue by proski** [siom79/jdrivesync#6]
The authentication file should not just be written to the current directory by default. The old UNIX way would be to put that file to the home directory. The XDG way would be to put it to `~/.local/share/jrivesync`. If cross-platform compatibility is desired, there are some ways to find the home directory in a portable way, see e.g. http://gamedev.stackexchange.com/questions/23040/is-there-a-cross-platform-special-directory-i-can-use-for-game-save-files

The problems with using the current directory are:
- It may be read-only.
- It may be exposed to other users.
- Users may not remember where the program was run the first time.

**Comment by siom79** [comment]
As a former grive user, I implemented it the same way as grive did it. But basically you are right. The user's home directory is a much better place as a default location. You can change this with the `-a` option anyhow.

I have committed a change set that now stores the authentication file in the user's home directory. You can test it with:

```
git clone https://github.com/siom79/jdrivesync.git
cd jdrivesync
git checkout develop
mvn clean install
```

Let me know what you think.

**Comment by proski** [comment]
I'm using that option in a wrapper around jdrivesync. But I think that jdrivesync should default to the sane behavior and it's the former grive users who should be using a wrapper :)

**Comment by siom79** [comment]
Released with version 0.3.0.
